src/python/grapl-ux-router/src/grapl_ux_router.py

```diff
@@ -87,9 +88,9 @@ def _retrieve_bucket_local(self) -> Bucket:
             try:
                 s3 = boto3.resource(
```

Comment:
If grapl_ux_router already consumes grapl-common, would be nice to use S3ResourceFactory.
Otherwise, whatever, it's okay

Reply:
I'll check on that.